src/basic/conf-files.h

```diff
@@ -29,3 +31,19 @@ int conf_files_list_dropins(
                 const char *dropin_dirname,
                 const char *root,
                 const char * const *dirs);
+
+typedef int parse_line_t(
+                void *data,
```

Comment:
we always call this "userdata", please do so here too.

it's a bit weird to put this arg first though, we always put that last